src/v/cluster/topic_table.h

```diff
@@ -262,6 +262,39 @@ class topic_table {
     using delta_cb_t = ss::noncopyable_function<void(delta_range_t)>;
     using lw_cb_t = ss::noncopyable_function<void()>;
 
+    // A helper struct that has various replica-related metadata all in one
+    // place, so that the API user doesn't have to query several maps manually.
+    struct partition_replicas_info {
```

Comment:
would be great to add short comment about the purpose of the `partition_replicas_info` in commit message